runtime/parser/expression.go

```diff
@@ -509,28 +532,47 @@ func init() {
 	defineExpr(postfixExpr{
 		tokenType:    lexer.TokenExclamationMark,
 		bindingPower: exprLeftBindingPowerUnaryPostfix,
-		leftDenotation: func(p *parser, left ast.Expression, tokenRange ast.Range) ast.Expression {
+		leftDenotation: func(p *parser, left ast.Expression, tokenRange ast.Range) (ast.Expression, error) {
 			return ast.NewForceExpression(
 				p.memoryGauge,
 				left,
 				tokenRange.EndPos,
-			)
+			), nil
 		},
 	})
 
 	defineNestedExpression()
-	defineInvocationExpression()
+	err = defineInvocationExpression()
```

Comment:
As noted above, these setup functions can just keep panicing, they'll never be called in a way where the error needs to be handled specifically